sdk/core/azure_core/perf.yml

```diff
@@ -34,15 +33,19 @@ parameters:
 - name: AdditionalArguments
   displayName: AdditionalArguments (passed to PerfAutomation)
   type: string
-  default: ' '
+  default: ''
 
 extends:
   template: /eng/pipelines/templates/jobs/perf.yml
   parameters:
-    ServiceDirectory: keyvault/azure_security_keyvault_secrets
+    ServiceDirectory: core/azure_core
     PackageVersions: ${{ parameters.PackageVersions }}
     Tests: ${{ parameters.Tests }}
     Arguments: ${{ parameters.Arguments }}
     Iterations: ${{ parameters.Iterations }}
     AdditionalArguments: ${{ parameters.AdditionalArguments }}
     Profile: ${{ parameters.Profile }}
+    EnvVars:
```

Comment:
We probably don't need this - it's a vestigal C++ value that I never got around to removing.